Gemfile.lock

```diff
@@ -56,6 +57,7 @@ DEPENDENCIES
   oj
   recog (>= 2.1.12)
   rspec (~> 3.1.0)
+  yard
 
 BUNDLED WITH
-   1.15.0
+   1.17.1
```

Comment:
I probably should manually revert this?

Reply:
Yes, I don't believe any of the Gemfile changes are required.